src/acom_music_box/music_box.py

```diff
@@ -229,7 +228,7 @@ def loadJson(self, path_to_json):
         camp_path = os.path.join(os.path.dirname(path_to_json), self.config_file)
 
         # Initalize the musica solver
-        self.solver = musica.create_solver(camp_path, musica.micmsolver.rosenbrock, 1)
+        self.solver = musica.create_solver(camp_path, musica.micmsolver.rosenbrock_standard_order, 1)
```

Comment:
Just curious, why has the solver type changed?

Reply:
The vector ordered solver has a default vector lenght of 4. It is the callers responsibility to appropriately size the input array to match what musica expects to write the results to. In this case, musica blindly copies the output values

https://github.com/NCAR/musica/blob/1debffba28d0601cbd646e475cd6751d06e5a8b3/src/micm/micm.cpp#L464-L465

So, if we have only 1 grid cell and 2 species, micm would internally hold a state array of length 8 (species 1 cell 1, specie 1 cell 2, specie 1 cell 3, species 1, cell 4, species 2 cell 1, species 2 cell 2, species 3 cell 3, species 4 cell 4). Thus, music box would need to correctly calculate the size of this array.

It's easiest to use the standard ordered solver because we don't have to care about those empty cells

Reply:
Ah, thank you for the detailed explanation!